app/controllers/application_controller.rb

```diff
@@ -1,4 +1,9 @@
 # frozen_string_literal: true
 
 class ApplicationController < ActionController::Base
+  private
+
+  def error_message(errors)
+    errors.full_messages.join(', ')
+  end
```

Comment:
Great use of the `ApplicationController` to have our other `Controllers` inherit the `error_message` method.